### PR TITLE
KAZOO-5289 Data views

### PR DESCRIPTION
**Pull request by lazedo**

### CODE REVIEW

File: applications/crossbar/src/crossbar_maintenance.erl [hunk 1]
```diff
@@ -1182,3 +1184,23 @@ set_app_screenshots(AppId, PathToScreenshotsFolder) ->
...
+
+-spec db_init() -> 'ok'.
+db_init() ->
+    kz_datamgr:suppress_change_notice(),
+    _ = kz_datamgr:revise_doc_from_file(?KZ_CONFIG_DB, ?APP, <<"views/system_configs.json">>),
+    _ = kz_datamgr:revise_doc_from_file(?KZ_MEDIA_DB, ?APP, "account/media.json"),
+    _ = kz_datamgr:revise_doc_from_file(?KZ_RATES_DB, ?APP, <<"views/rates.json">>),
+    _ = kz_datamgr:revise_doc_from_file(?KZ_SIP_DB, ?APP, <<"views/resources.json">>),
+    _ = kz_datamgr:revise_doc_from_file(?KZ_PORT_REQUESTS_DB, ?APP, <<"views/port_requests.json">>),
+    _ = kz_datamgr:revise_doc_from_file(?KZ_ACDC_DB, ?APP, <<"views/acdc.json">>),
```
Comment (jamesaimonetti) [inline applications/crossbar/src/crossbar_maintenance.erl]: I am not sure what to do about community apps having their assets referenced in kazoo's core apps. Like, when someone creates the next 'foobar' app with account views, does Crossbar now need a line added to load the foobar app's views and foobar needs to put their views in crossbar/priv?

File: core/kazoo_data/src/kz_datamgr.erl [hunk 2]
```diff
@@ -194,16 +202,28 @@ do_revise_docs_from_folder(DbName, Sleep, [H|T]) ->
...
 
+maybe_update_doc(DbName, JObj) ->
+    case should_update(DbName, JObj) of
+        true -> ensure_saved(DbName, JObj);
+        false -> {'ok', JObj}
+    end.
+
+should_update(DbName, JObj) ->
+    case open_doc(DbName, kz_doc:id(JObj)) of
+        {'ok', Doc} -> not kz_json:are_equal(JObj, kz_doc:delete_revision(Doc));
```
Comment (jamesaimonetti) [inline core/kazoo_data/src/kz_datamgr.erl]: Should it compare public fields only?

File: applications/crossbar/src/crossbar_maintenance.erl (hunk 1 above)
Comment (lazedo) [reply applications/crossbar/src/crossbar_maintenance.erl]: these were already there, but could/should be moved to the app that is responsible for it.
these apps should register/revide on app startup

File: core/kazoo_data/src/kz_datamgr.erl (hunk 2 above)
Comment (lazedo) [reply core/kazoo_data/src/kz_datamgr.erl]: maybe

File: applications/crossbar/src/crossbar_maintenance.erl [hunk 3]
```diff
@@ -1182,3 +1184,23 @@ set_app_screenshots(AppId, PathToScreenshotsFolder) ->
               || SShot <- filelib:wildcard(kz_term:to_list(PathToScreenshotsFolder) ++ "/*.png")
              ],
     update_screenshots(AppId, MA, SShots).
+
+-spec db_init() -> 'ok'.
+db_init() ->
+    kz_datamgr:suppress_change_notice(),
+    _ = kz_datamgr:revise_doc_from_file(?KZ_CONFIG_DB, ?APP, <<"views/system_configs.json">>),
+    _ = kz_datamgr:revise_doc_from_file(?KZ_MEDIA_DB, ?APP, "account/media.json"),
```
Comment (kalda341) [inline applications/crossbar/src/crossbar_maintenance.erl]: Meant to be <<"account/media.json">> ?

Comment (lazedo) [reply applications/crossbar/src/crossbar_maintenance.erl]: same thing. `kz_datamgr:revise_doc_from_file` does a `list_to_binary`

Comment (k-anderson) [reply applications/crossbar/src/crossbar_maintenance.erl]: @lazedo for consistency I would like these to match the datatype :)

File: core/kazoo_apps/src/kapps_maintenance.erl [hunk 4]
```diff
@@ -85,8 +87,9 @@ unbind(Event, M, F) -> kazoo_bindings:unbind(binding(Event), M, F).
 
 -spec refresh_account_db(ne_binary()) -> 'ok'.
 refresh_account_db(Database) ->
-    Classification = 'account' = kz_datamgr:db_classification(Database),
-    kapi_maintenance:refresh_views(Database, Classification),
+    %%     Classification = 'account' = kz_datamgr:db_classification(Database),
```
Comment (k-anderson) [inline core/kazoo_apps/src/kapps_maintenance.erl]: @lazedo Please remove dead code :)

**Pull request merged**